modules/consensus/accept.go

```diff
@@ -19,6 +19,21 @@ var (
 	errOrphan          = errors.New("block has no known parent")
 )
 
+// managedBroadcastBlock will broadcast a block to the consensus set's peers.
+func (cs *ConsensusSet) managedBroadcastBlock(b types.Block) {
+	// COMPATv0.5.1 - broadcast the block to all peers <= v0.5.1 and block header to all peers > v0.5.1.
```

Comment:
I almost removed it. But it's a separate concern for a different PR. 

It's probably prudent to wait a little longer.